devices/JointTorqueControlDevice/src/PassThroughControlBoard.cpp

```diff
@@ -0,0 +1,1537 @@
+/**
+ * @file PassThroughControlBoard.cpp
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#include <BipedalLocomotion/PassThroughControlBoard.h>
+#include <iostream>
+#include <yarp/os/Property.h>
+
+using namespace yarp::dev;
+using namespace BipedalLocomotion;
+
+// CONSTRUCTOR
+PassThroughControlBoard::PassThroughControlBoard()
+    : proxyIEncodersTimed(0)
+    , proxyIPositionControl(0)
+    , proxyIVelocityControl(0)
+    , proxyIPositionDirect(0)
+    , proxyIControlMode(0)
+    , proxyITorqueControl(0)
+    , proxyIOpenLoopControl(0)
+    , proxyIInteractionMode(0)
+    , proxyIPidControl(0)
+    , proxyICurrentControl(0)
+    , proxyIMotorEncoders(0)
+    , proxyIAxisInfo(0)
+    , proxyIAmplifierControl(0)
+    , proxyIControlCalibration(0)
+    , proxyIControlLimits(0)
+    , proxyIMotor(0)
+{
+}
+
+PassThroughControlBoard::~PassThroughControlBoard()
+{
+    this->close();
+}
+
+void PassThroughControlBoard::resetPointers()
+{
+    proxyIEncodersTimed = nullptr;
+    proxyIPositionControl = nullptr;
+    proxyIControlMode = nullptr;
+    proxyITorqueControl = nullptr;
+    proxyIOpenLoopControl = nullptr;
+    proxyIVelocityControl = nullptr;
+    proxyIPositionDirect = nullptr;
+    proxyIInteractionMode = nullptr;
+    proxyIPidControl = nullptr;
+    proxyICurrentControl = nullptr;
+    proxyIMotorEncoders = nullptr;
+    proxyIAxisInfo = nullptr;
+    proxyIAmplifierControl = nullptr;
+    proxyIControlCalibration = nullptr;
+    proxyIControlLimits = nullptr;
+    proxyIMotor = nullptr;
+}
+
+// DEVICE DRIVER
+bool PassThroughControlBoard::open(yarp::os::Searchable& config)
+{
+    bool legacyOption = false;
+    legacyOption = legacyOption || config.check("proxy_remote");
+    legacyOption = legacyOption || config.check("proxy_local");
+
+    if (legacyOption)
+    {
+        yError("PassThroughControlBoard error: legacy option proxy_remote and proxy_local are not "
+               "supported anymore. Please use the yarprobotinterface to compose devices.\n");
+        return false;
+    }
```

Comment:
I think this can be removed.

What do you think @traversaro ?

Reply:
Yes, it was a leftover from the copy&paste from controlboardwrapper2 or similar I guess.